protocol/simplified-guardian.md

```diff
@@ -0,0 +1,118 @@
+when this definition goes into effect.
+
+## Proposed Solution
+
+### Remove the Deputy Guardian
+
+We begin by removing the Deputy Guardian role entirely. The Guardian role would be held exclusively
+by the 1/1 Security Council Operations Safe (held by the Security Council). All Guardian actions
+would therefore only be executable by a majority of the Security Council. This eliminates the
+intial problem posed by the updated Stage 1 definition.
+
+### Deputy Pause Module v2
+
+We still want the Optimism Foundation to retain the power to quickly respond to potential
+incidents. We would therefore install a modified version of the Deputy Pause Module into the
+Security Council Operations Safe and permit the Optimism Foundation Safe (or whatever party
+previously held the Deputy Guardian role for that system prior to this change) to manage the Pause
+Deputy account. This permits the Pause Deputy to trigger the pause mechanism freely.
+
+### Pause Expiry
+
+Because the Pause Deputy could trigger the pause mechanism at any time and the unpause function
+would require participation from a majority of the Security Council, we must either (1) allow a
+minority of the Security Council to unpause the system or (2) have the pause automatically expire
+after some period of time. Given the potential security concerns with allowing a minority of the
+Security Council to unpause, we prefer option (2).
+
+We therefore propose that the pause would expire after 6 months. Once the pause is used, the pause
+cannot be called again unless it is reset by a majority of the Security Council (this applies in
+all cases, regardless of whether the system unpauses due to expiry or due to explicit unpause).
+
+### Cluster-wide Pause
+
+Because the pause mechanism would become the main mechanism available to the Optimism Foundation,
+the pause mechanism would also become the basis for an updated incident response process. However,
+the Superchain-wide Pause is far too impactful to be the only available option during an incident.
+Given that we suspect many incidents will take the form of *single* impacted chains or chain
+clusters, it may be valuable to implement a Cluster-wide Pause on top of the Superchain-wide Pause.
+
+A chain would be paused if *either* the Superchain-wide Pause is active or its Cluster-wide Pause
+is active. We would implement the Cluster-wide Pause by allowing the Guardian to pause a specific
+chain/cluster as identified by a `SystemConfig` contract address. That is, the Guardian would call
+`SuperchainConfig.pause(someSystemConfigAddress)`. Other contracts would then check the status of
+the pause by calling `SuperchainConfig.paused(mySystemConfigAddress)` which would return true if
+either that specific system is paused or the entire Superchain is paused.
+
+The interop cluster can use a "fake" shared system config address in the call to `paused(..)` so
+that the entire cluster pauses at the same time.
```

Comment:
from this snippet, I gather that each OP Chain will need to store `mySystemConfigAddress` somewhere, and that having the same value defines a cluster? 

How will this be kept in sync if a chain moves into a cluster from having been on its own?

Would it be better to use the lockbox address for this, since it is already intended to eventually become a singleton per cluster?

Reply:
I'm on board with that general direction. 
Since there is already a reference to the SuperchainConfig in the OptimismPortal, it would probably be easier and less diff to do: 

2. Contracts check `OptimismPortal.paused()` which passes the lockbox address as the pause identifier to `SuperchainConfig.paused(identifier)`.

Reply:
I think it's cleaner if the SystemConfig is the source of truth here instead of the OptimismPortal. I think the diff will look the same either way.